category/category.go

```diff
@@ -162,7 +163,7 @@ func (m *GormRepository) LoadAllRelationshipsOfCategory(ctx context.Context, cat
 		log.Error(ctx, map[string]interface{}{
 			"category_id": categoryID,
 		}, "category not found")
```

Comment:
That error message is wrong as well. You cannot tell if the category wasn't found or if there was an internal error!